attributes/default.rb

```diff
@@ -19,3 +22,6 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
+
+default['docker_machine']['version'] = 'v0.14.0'
```

Comment:
remove all these stuff, not useful to users and not supposed to be used by users.